test/index.test.js

```diff
@@ -1273,4 +1275,85 @@ Previous tag: ''
       )
     })
   })
+
+  describe('with config-name input', () => {
+    it('loads from another config path', async () => {
+      /*
+        Mock
+        with:
+          config-name: 'config-name-input.yml'
+      */
+      process.env['INPUT_CONFIG-NAME'] = 'config-name-input.yml'
+
+      // Mock config request for file 'config-name-input.yml'
+      const getConfigScope = nock('https://api.github.com')
+        .get(
+          '/repos/toolmantim/release-drafter-test-project/contents/.github/config-name-input.yml'
+        )
+        .reply(200, {
+          type: 'file',
+          encoding: 'base64',
+          size: 5362,
+          name: 'config-name-input.yml',
+          path: '.github/config-name-input.yml',
+          content: encodeContent(
```

Comment:
use `getConfigMock` and rewrite it serve the purpose and avoiding the `release-drafter.yml` ?
Or Write a new function.